custom_components/ha_creality_ws/__init__.py

```
@@ -418,35 +427,46 @@ async def _state_cb(event) -> None:
     return True
 
 
+def _coordinators_for_devices(hass: HomeAssistant, device_ids: Any) -> list:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `device_ids`

(ANN401)